indexer/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM rust:1.62 AS builder
+WORKDIR /tmp/
+COPY Cargo.toml Cargo.lock ./
+COPY database database
+COPY indexer indexer
+# Copy 'circulating-supply' because is member of the workspace but is not compiled or added to the
+# output image.
+COPY circulating-supply circulating-supply
+RUN cargo build -p indexer-explorer-lake --release
+
+FROM ubuntu:20.04
+RUN apt update && apt install -yy openssl ca-certificates
+RUN apt-get install libpq5 -y
+COPY --from=builder /tmp/target/release/indexer-explorer-lake .
+CMD [ "./indexer-explorer-lake" ]
```

```suggestion
CMD [ "./indexer-explorer-lake",  $CHAIN_ID, $START_BLOCK_OPTIONS]
```
For now - can we add these here to get around `indexer`s inability to take chain/block options as environment variables?

---

Ignore this - we can just pass command line arguments to the docker image itself